cli/wsproxy.ml

```diff
@@ -13,17 +13,14 @@
  *)
 open Wslib
 
-let get_dir_path () = Printf.sprintf "/var/xapi/" 
-
 module LwtWsIteratee = Wslib.Websockets.Wsprotocol(Lwt)
 open Lwt.Infix
 
 let with_fd = Lwt_support.with_fd
 
-let start path handler =
-  let dir_path = get_dir_path () in
-  let fd_sock_path = Printf.sprintf "%s%s" dir_path path in
-  Lwt_log.info_f "Starting wsproxy on %s" fd_sock_path >>= fun () ->
+let start handler =
+  let fd_sock_path = "/var/xapi/wsproxy" in
+  Logs_lwt.info (fun m -> m "Starting wsproxy on %s" fd_sock_path) >>= fun () ->
   let fd_sock = Lwt_unix.socket Unix.PF_UNIX Unix.SOCK_STREAM 0 in
```

Comment:
Can we use Lwt_unix.stdin here?

Reply:
yes, haven't had time to push the change ;p